src/mantine-hooks/src/use-viewport-size/use-viewport-size.ts

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useState, useEffect } from 'react';
+import { useWindowEvent } from '../use-window-event/use-window-event';
+
+interface DebouncedWindowSizeOptions {
```

Comment:
gonna fix the name after review

Reply:
Just remove it, hook should not receive any values, there is no point to do default value